ui/library/src/context/PageRenderContext.ts

```diff
@@ -82,6 +87,16 @@ export function usePageRenderContextProps({
     [pageRenderStates]
   );
 
+  const isFinishedBuildingAllPagesObjectURLs = React.useCallback((): boolean => {
+    if (!pdfDocProxy) return false;
+    for (let pageNumber = 1; pageNumber <= pdfDocProxy.numPages; pageNumber++) {
+      if (!pageRenderStates.get(pageNumber)?.objectURL) {
```

Comment:
@ericmarsh995 instead of doing this u can call `isBuildingObjectURLForPage()` and passing in the page number

Reply:
I would do this, but isBuildingObjectURLForPage() returns false when webite first loads (because its not actually building the objectURLS). isFinishedBuildingAllPagesObjectURLs() should only return true once all pages are loaded